src/framework/jobState/FileSystemJobState/indices.ts

```diff
@@ -0,0 +1,89 @@
+import { Entity, Relationship } from '../../types';
+import { GraphObjectIteratee } from '../types';
+
+import {
+  walkDirectory,
+  WalkDirectoryIterateeInput,
+} from '../../../cacheDirectory';
```

Comment:
Is this in `src/cacheDirectory` because you anticipate using it in code outside of `jobState/` or `FileSystemJobState/`?

Reply:
Yeah, hope to use it when performing `synchronization` (most of that code will likely live under it's directory).